raisinbread/RB_files/RB_electrophysiology_uploader.sql

```diff
@@ -0,0 +1,14 @@
+INSERT INTO `electrophysiology_uploader` (`UploadID`,`MetaData`,`UploadDate`,`SessionID`,`UploadLocation`,`Status`,`UploadedBy`) 
```

Comment:
insert "SET FOREIGN_KEY_CHECKS=0;
TRUNCATE TABLE `electrophysiology_uploader `;
LOCK TABLES `electrophysiology_uploader ` WRITE; " in first line.